# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lcap/hotfixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./lcap.c.orig	2003-02-10 11:52:30.000000000 +0100
+++ ./lcap.c	2003-02-10 11:52:41.000000000 +0100
@@ -27,6 +27,7 @@
 #include <getopt.h>
 #include <stdlib.h>
 #include <ctype.h>
+#include <limits.h>
 
 #define PROC_CAP "/proc/sys/kernel/cap-bound"
 
